python/ray/_private/structured_logging/formatters.py

```diff
@@ -1,24 +1,53 @@
 import logging
 import json
 from ray._private.structured_logging.constants import LogKey, LOGRECORD_STANDARD_ATTRS
+from ray._private.ray_constants import LOGGER_FORMAT
 
 
-class JSONFormatter(logging.Formatter):
-    def format(self, record):
-        record_format = {
-            LogKey.ASCTIME: self.formatTime(record),
+def generate_record_format_attrs(
+    formatter: logging.Formatter,
+    record: logging.LogRecord,
+    exclude_standard_attrs=False,
```

Comment:
I have decided to use `exclude_standard_attrs` instead of `include_standard_attrs` or `include_extra_attrs` because I think `exclude_standard_attrs` is more accurate.

* `include_standard_attrs`: This function doesn't include all standard LogRecrod attributes.
* `include_extra_attrs`: This function not only include attributes specified by `logger.info(..., extra={...})` but also Ray context.